.github/workflows/openshift-upgrade-test.yaml

```
@@ -33,6 +33,8 @@ jobs:
   e2e-tests:
     name: Prepare E2E configuration and image
     runs-on: ubuntu-latest
+    environment: test
+    if: vars.SKIP_OPENSHIFT != 'true'
```

Maybe it is `env.` ?

```suggestion
    if: env.SKIP_OPENSHIFT != 'true'
```